build/pipeline-uiautomation.yaml

```diff
@@ -0,0 +1,249 @@
+
+# UI Automation builds should be triggered on every branch except master (so we don't trigger again after a merge)
+trigger:
+  branches:
+    include:
+    - '*'
+    exclude:
+    - master
+
+variables:
+  BuildPlatform: 'any cpu'
+  BuildConfiguration: 'debug'
+  FastRun: true
+
+#BUILD PHASE
```

Comment:
Have you verified that the build phases and the app center phases run in parallel?

Reply:
yes